.travis.yml

```diff
@@ -1,21 +1,38 @@
 language: python
-python: 2.7
 sudo: false
+matrix:
+  include:
+    - python: "2.7"
+      env: PLONE_VERSION=4.3.x
+    - python: "2.7"
+      env: PLONE_VERSION=5.0.x
+    - python: "2.7"
+      env: PLONE_VERSION=5.1.x
+    - python: "2.7"
+      env: PLONE_VERSION=5.2.x
+    - python: "3.6"
+      env: PLONE_VERSION=5.2.x
+    - python: "3.7"
+      env: PLONE_VERSION=5.2.x
+      dist: xenial
+      sudo: true
+  fast_finish: true
 cache:
   directories:
   - eggs
   - parts/node
   - downloads
-env:
-- PLONE_VERSION=4.3
-- PLONE_VERSION=5.0
-- PLONE_VERSION=5.1
-matrix:
-  fast_finish: true
+  - $HOME/buildout-cache
+before_install:
+  - mkdir -p $HOME/buildout-cache/{eggs,downloads}
+  - mkdir $HOME/.buildout
+  - echo "[buildout]" > $HOME/.buildout/default.cfg
+  - echo "download-cache = $HOME/buildout-cache/downloads" >> $HOME/.buildout/default.cfg
+  - echo "eggs-directory = $HOME/buildout-cache/eggs" >> $HOME/.buildout/default.cfg
+  - pip install zc.buildout
```

Comment:
shouldn't that be `pip install -r requirements.txt` ?

Reply:
Thanks for the great feedback! I fully agree with you on this one. It was copy and pasted from another add-on and did not see the problem. I'll submit a fix to the other repo as well.